lib/auth/sessions.go

```diff
@@ -277,6 +279,14 @@ type NewAppSessionRequest struct {
 	MFAVerified string
 	// DeviceExtensions holds device-aware user certificate extensions.
 	DeviceExtensions DeviceExtensions
+	// AppName is the name of the app.
+	AppName string
+	// AppURI is the URI of the app.
+	AppURI string
```

Comment:
It's not immediately clear what the difference between the PublicAddr and URI field - we should probably make sure this is clear here and in RouteToApp.

Reply:
+1